hack/tests/e2e-ansible-molecule.sh

```diff
@@ -8,8 +8,15 @@ ROOTDIR="$(pwd)"
 GOTMP="$(mktemp -d)"
 trap_add 'rm -rf $GOTMP' EXIT
 # Needs to be from source until 2.20 comes out
-pip install --user git+https://github.com/ansible/molecule.git
-pip install --user docker openshift jmespath
+pip install --user \
+    pyasn1==0.4.5 \
+    pyasn1-modules==1.0.22
+pip install --user --ignore-installed \
+    git+https://github.com/ansible/molecule.git
```

Comment:
This we can pin to 2.20, this is likely the cause of our issues. It was initially installed from source because we relied on a few patches that hadn't yet made it to a release